faq/README.md

```diff
@@ -88,6 +88,17 @@ session.execute(batch);
 ```
 
 
+### Why do my 'select *' `PreparedStatement`-based queries stop working after a schema change?
```

Comment:
I see that we tend to use upper case letters for CQL keywords throughout the manual, so for consistency I suggest changing 'select *' to `SELECT *`.